shared/deployments/check-publish.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+git diff --quiet HEAD~ HEAD -- $PWD/contracts
+
+if [ $? == "1" ]
+then
+  aragon apm publish major --files dist/ --environment continuous-deployment --apm.ipfs.rpc https://ipfs.autark.xyz:5001 --ipfs-check false --propagate-content false --skip-confirmation --no-prepublish
+  exit $?
+else 
+  echo "no contract changes"
+fi
+
+git diff --quiet HEAD~ HEAD -- $PWD/app $PWD/../../shared/identity $PWD/../../shared/lib $PWD/../../shared/store-utils $PWD/../../shared/ui $PWD/../../shared/utils
+
+if [ $? == "1" ]
+then
+  aragon apm publish minor --files dist/ --environment continuous-deployment --apm.ipfs.rpc https://ipfs.autark.xyz:5001 --ipfs-check false --propagate-content false --skip-confirmation
+  exit $?
```

Comment:
Why exit with an error status?

Reply:
This exits with the error status of the publish command. If it's zero, it will exit with zero. Otherwise it'll throw an error and Travis will reflect the result. The current publish script show that the publish was successful even if an app errors on publishing, which isn't very helpful